js/components/cohort-definitions/cohort-definition-manager.js

```diff
@@ -574,6 +583,19 @@ define(['knockout', 'text!./cohort-definition-manager.html',
 			self.closeConceptSet();
 		}
 
+		self.showSaveConceptSet = function() {
+		  self.newConceptSetName(self.model.currentConceptSet().name());
+		  self.saveConceptSetShow(true);
+		};
+
+		self.hideSaveConceptSet = function() {
+		  self.saveConceptSetShow(false);
+		};
+
+		self.saveConceptSet = function () {
+      self.saveConceptSetShow(false);
```

Comment:
In testing, I found that the new 'Save Concept Set' button was not copying the concept set back to the repository. I traced it back to here and it appears that perhaps the implementation is incomplete?